package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-smart-tabs",
-  "version": "0.0.15",
+  "version": "0.0.16",
```

Comment:
Is this a feature or a fix? If this is a feature so please use `0.1.0` Version.

Reply:
done